pytorch_lightning/loggers/tensorboard.py

```diff
@@ -173,3 +175,15 @@ def _get_next_version(self):
             return 0
 
         return max(existing_versions) + 1
+
+    def _sanitize_params(self, params):
+        native_types = [int, bool, float, str, torch.Tensor]
+        out_dict = {}
+
+        for k, v in params.items():
+            if type(v) not in native_types:
+                out_dict[k] = repr(v)
+            else:
+                out_dict[k] = v
```

Comment:
```suggestion
            out_dict[k] = repr(v) if type(v) not in native_types else v
```

Reply:
just thinking, `str()` and `repr()` are the same mostly...

Reply:
Ya, I had originally wrote repr, when we were discussing the eval method of decoding. str is more canonical so will switch